locales/base/translation.json

```diff
@@ -1375,11 +1375,11 @@
   },
   "fiatConnectStatusScreen": {
     "tryAgain": "Try Again",
-    "withdraw": {
```

Comment:
don't we have to keep this around for existing apps in the wild that will try to access `.withdraw.`?